### DISCUSSION TITLE
Renew Rustdesk Let's Encrypt certificate after expiring (2)

### DISCUSSION THREAD

**Discussion by Affection4036** [rustdesk/rustdesk#9822] [Q&A] [answered]
Hello,

Last month I created a topic to ask for help on how to manually renew the Let's Encrypt certificate when using the Pro version. Unfortunately my server doesn't automatically renew the certificate itself and now it has been expired. What can I do to renew this certificate? [In my earlier post](https://github.com/rustdesk/rustdesk/discussions/9661) a comment was made on how to renew my Pro license, but this is not what I meant as I still have several months remaining on my current license.

**Comment by rustdesk** [comment, marked as answer]
https://docs.digitalocean.com/support/how-can-i-renew-lets-encrypt-certificates/

**Reply by Affection4036** [reply]
Running `sudo certbot renew` was indeed the answer, just wanted to make sure I did it the right way. Thanks a lot!
